<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Bus</class>
 <widget class="QDialog" name="Bus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Bus</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>621</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_Ajouter">
    <attribute name="title">
     <string>Ajouter Bus</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_okAjout">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>320</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom chauffeur</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_IdBus">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Disponibilité</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_dispo">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>240</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>oui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>non</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Bus</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Matricule">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Marque">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Capacite">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Chauffeur">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Etat">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>150</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>En panne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En cours de réparation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En service</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Libre</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="labelB">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB</zorder>
    <zorder>pushButton_okAjout</zorder>
    <zorder>label_4</zorder>
    <zorder>label_2</zorder>
    <zorder>label_6</zorder>
    <zorder>lineEdit_IdBus</zorder>
    <zorder>label_7</zorder>
    <zorder>comboBox_dispo</zorder>
    <zorder>label</zorder>
    <zorder>lineEdit_Matricule</zorder>
    <zorder>label_3</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_Marque</zorder>
    <zorder>lineEdit_Capacite</zorder>
    <zorder>lineEdit_Chauffeur</zorder>
    <zorder>comboBox_Etat</zorder>
   </widget>
   <widget class="QWidget" name="tab_Afficher">
    <attribute name="title">
     <string>Afficher Bus</string>
    </attribute>
    <widget class="QTableView" name="tabBUS">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_Supprimer">
    <attribute name="title">
     <string>Supprimer Bus</string>
    </attribute>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Bus</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercherSupp">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>100</y>
       <width>93</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chercherSupp">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>431</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_2</zorder>
    <zorder>label_8</zorder>
    <zorder>pushButton_chercherSupp</zorder>
    <zorder>lineEdit_chercherSupp</zorder>
   </widget>
   <widget class="QWidget" name="tab_Modifier">
    <attribute name="title">
     <string>Modifier Bus</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Matricule</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_Etat">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Marque</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_chercherModif">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>93</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_NomChauf">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>260</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Modif_dispo">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>300</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>oui</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>non</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_Marque">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>140</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_chercherModif">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Modif_Etat">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>180</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>En panne</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En cours de réparation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>En service</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Libre</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Disponibilité</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Bus</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom chauffeur</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Modif_Confirmer">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>370</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_Capacite">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_Matricule">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>100</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>220</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Capacité</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Modif_Dispo">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>300</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelB_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>611</width>
       <height>431</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>labelB_3</zorder>
    <zorder>label_9</zorder>
    <zorder>lineEdit_Modif_Etat</zorder>
    <zorder>label_10</zorder>
    <zorder>pushButton_chercherModif</zorder>
    <zorder>lineEdit_Modif_NomChauf</zorder>
    <zorder>comboBox_Modif_dispo</zorder>
    <zorder>lineEdit_Modif_Marque</zorder>
    <zorder>lineEdit_chercherModif</zorder>
    <zorder>comboBox_Modif_Etat</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>pushButton_Modif_Confirmer</zorder>
    <zorder>lineEdit_Modif_Capacite</zorder>
    <zorder>lineEdit_Modif_Matricule</zorder>
    <zorder>label_14</zorder>
    <zorder>label_15</zorder>
    <zorder>lineEdit_Modif_Dispo</zorder>
   </widget>
   <widget class="QWidget" name="tab_Stat">
    <attribute name="title">
     <string>Statistique</string>
    </attribute>
    <widget class="QTableView" name="tabBUS_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
